data/hai/hai reveal 5 philosophers.txt

```diff
@@ -631,124 +493,98 @@ fleet "Smuggler (Hai Traffickers)"
 	names "civilian"
 	cargo 0
 	personality timid
-	variant 3
+	variant 10
 		"Blackbird (Hai Trafficker)"
-	variant 2
+	variant 5
+		"Bulk Freighter (Hai Trafficker)"
```

Comment:
These seem much easier to catch than any other ship on the list.

Reply:
Yes, it is gone now.